sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -66,6 +66,12 @@ private[spark] object HiveUtils extends Logging {
     .stringConf
     .createWithDefault(builtinHiveVersion)
 
+  // A deprecated config which is only used to provide a default value, in case some existing
+  // applications depend on this config, e.g. Spark SQL ODBC driver.
+  val HIVE_EXECUTION_VERSION = buildConf("spark.sql.hive.version")
```

Comment:
doc may needed

Reply:
This config is a kind of a fake one, I think comments are enough for it.

Reply:
or maybe more meaningful name